test/src/test/java/hudson/security/csrf/CrumbExclusionTest.java

```diff
@@ -81,8 +67,16 @@ public void pathInfo() throws Exception {
             try {
                 fail(path + " should have been rejected: " + r.createWebClient().login("admin").getPage(new WebRequest(new URL(r.getURL(), path + "?script=11*11"), HttpMethod.POST)).getWebResponse().getContentAsString());
             } catch (FailingHttpStatusCodeException x) {
-                assertEquals("status code using " + path, 403, x.getStatusCode());
-                assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("No valid crumb was included in the request"));
+                switch (x.getStatusCode()) {
+                case 403:
+                    assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("No valid crumb was included in the request"));
+                    break;
+                case 400: // from Jetty
+                    assertThat("error message using " + path, x.getResponse().getContentAsString(), containsString("Ambiguous path parameter in URI"));
```

Comment:
Or could just delete the `/..;/cli` branch, which is no longer really testing anything interesting here.